src/main/cpp/state/RobotStateChanges.h

```diff
@@ -30,6 +30,7 @@ class RobotStateChanges
         CompressorChange_Int,
         ChassisPose_Pose2D,
         LoopCounterEnd,
+        ElevatorHeight_Int,
```

Comment:
This should be ElevatorHeight_Inch and you need to publish a inch value not a double